Taskfile.yml

```diff
@@ -228,6 +228,7 @@ tasks:
     cmds:
       - bin/golangci-lint{{exeExt}} run --config=.golangci-new.yml
       - go vet -vettool=./bin/checkswitch{{exeExt}} ./...
+      - bin/task{{exeExt}} -d integration lint
```

Comment:
That should be dependency in the `deps` section above. Dependencies are run in parallel, so `task lint` works faster.

On that line we should check `.golangci-new.yml` rules for integration module